features/importing_service_definitions.feature

```diff
@@ -10,7 +10,7 @@ Feature: Importing service definitions
       extensions:
         Zalas\Behat\NoExtension:
           imports:
-            - features/bootstrap/config/services.yml
+            - %paths.base%/features/bootstrap/config/services.yml
```

Comment:
> what if someone tries to use other Behat parameters?
I thought about that. I also thought to get there iteratively :)

> I guess I'm fine with adding the fix, but I'd keep this example as it was. 
Oki. Would you rather add a scenario or an entire feature for that?

> %paths.base% is technically not needed as everything is resolved relative to this path
True. I added it there to show it's possible to use it.